src/cloud/env/variables-deploy.md

```diff
@@ -408,6 +408,9 @@ If you specify `\Magento\Framework\Cache\Backend\RemoteSynchronizedCache` as the
 -  **Default**—`false`
 -  **Version**—Magento 2.1.16 and later
 
+{:.bs-callout-warning}
+You should not enable this on scaled architecture (split architecture) projects because it will cause Redis connection errors. Redis slaves are still active but will not be used for Redis reads. As an alternative for scaled architecture, we recommend using Magento 2.3.5 or later, implementing a new Redis backend configuration, and implementing L2 caching for Redis.
```

Comment:
```suggestion
Do not enable this variable on scaled architecture (split architecture) projects because it will cause Redis connection errors. Redis slaves are still active but will not be used for Redis reads. As an alternative, we recommend the following: use Magento 2.3.5 or later on Cloud projects with a scaled architecture, implement a new Redis backend configuration, and implement L2 caching for Redis.
```

Reply:
Change "will cause" to "causes". 
If you **do** it, it doesn't will cause, it causes.